js/modules/k6/ws/ws.go

```diff
@@ -667,3 +620,87 @@ func wrapHTTPResponse(httpResponse *http.Response) (*HTTPResponse, error) {
 
 	return &wsResponse, nil
 }
+
+//nolint:gocognit
```

Comment:
What should we do for removing it? Would adding a default case in the switch statement be enough?

Reply:
Doesn't seem so. We would likely need to extract some of it into helper functions, but IMO this function is readable enough, and it's only slightly over the linter limit (33 > 30). At a certain point, there are diminishing returns trying to appease a linter, so I wouldn't bother with it. Especially since we plan to deprecate this module anyway. :man_shrugging:

Reply:
LGTM